### DISCUSSION TITLE
Design System

### DISCUSSION THREAD

**Discussion by poyribn** [tenacityteam/tenacity-legacy#487] [General]
> @Bitsper2nd wrote in [this comment](https://github.com/tenacityteam/tenacity/discussions/159#discussioncomment-1185712):
> The first step would be to actually do a Design System for Tenacity.

A Design System is a documentation of design elements
for making collaborative design easier and consistent.

þis thread is for shaping þe tenacity design system, because we don't have one yet.

**Comment by poyribn** [comment]
Let's start with þe colors, because any good design system has good colors.
¿How about þese ones based on [my tenacity webpage design](https://web.archive.org/web/20210809195638/https://sosasees.github.io/tenacity-webpage/)?:
![suggested colors palette](https://user-images.githubusercontent.com/66305550/129678401-5f3e1276-9945-40d2-983e-949cb8f729d6.png)

**Reply by bitsper2nd** [reply]
Well here is you're my feedback. This time with some visuals to help you and anyone else reading this. I will start with saying; Good job on the color scheme! Color are greats and have nice contrast. This was tested on Firefox.

#### light theme
![tenacity-light](https://user-images.githubusercontent.com/22452474/129746750-32b2cdf7-b4e3-44d1-b264-6b4dafa86482.png)

#### dark theme
![tenacity-dark](https://user-images.githubusercontent.com/22452474/129746793-77592017-96a0-421a-9add-47ddb3f1731c.png)

Here is how the site looks on a phone and tablet:

#### iphone X
![Screenshot 2021-08-17 at 10-27-38 Tenacity](https://user-images.githubusercontent.com/22452474/129747431-fbc0d91f-3e3c-4275-bae4-523e027f3a8a.png)


#### ipad
![Screenshot 2021-08-17 at 10-28-14 Tenacity](https://user-images.githubusercontent.com/22452474/129747946-5b0c1d61-2ef2-42fc-9508-9a331918777a.png)

Now you may wonder as to why I am posting screenshots of the site? Well is to compare them to the desktop. 

#### Desktop
![Screenshot 2021-08-17 at 10-26-57 Tenacity](https://user-images.githubusercontent.com/22452474/129748503-b2f29bea-91b1-4fb1-8cb5-efed88c58e74.png)

As you can see there is a lot of wasted space on this version of the site. One easy fix for this would be to make content below the header use two columns. 

#### Buttons
![buttons](https://user-images.githubusercontent.com/22452474/129746285-487e1417-4447-48cd-8e13-d086e3494f7d.png)

Finally, the buttons in the documentation are inconsistent. They should have the same width and height. Some of those buttons could serve better in a navigation bar. This is all the feedback I have for now. Hope you can find it helpful. 

**PS:** Not really relevant, but just in case someone is curious. The system font you see on the screenshot is called Metro Nova.

**Reply by poyribn** [reply]
Thank you very much for the suggestions.
I have put them into [my webpage design](https://web.archive.org/web/20210821221155/https://sosasees.github.io/tenacity-webpage/).

These are the relevant changes:
- Moved the "wall of buttons" to the bottom of the page,
repeated the most important buttons in a top bar
- Resized the buttons on the "wall of buttons" to be either full-witdh or half-width
with no inbetwens like before
- Changed the "features list" from a simple list to a responsive grid
- Limited the size of the screenshot to further remove the perception of "too much whitespace".
Whitespace is only too much if it's much more than on most of the website.
Previously, the screenshot being much wider than anything else
created the perception of a lot of unused space on the left and right sides of the page.

**Reply by n0toose** [reply]
Feedback gathered from Mastodon and Twitter:
- [not a huge fan of the max text width, feel that it should be a bit bigger on desktop, I really like the Features section tho](https://fosstodon.org/web/statuses/106800349476106236)
- [Looks better on mobile. (Followed by an "I agree")](https://fosstodon.org/web/statuses/106800321241677797)
- [Really like seeing the download, docs, and source code links up top. Projects tend not to do that anymore, this is refreshing. I'd say, personally I'd prefer to see more text there and a couple more screenshots, but all in thumbnail form, below some of the useful text that's now below the large screenshot.](https://fosstodon.org/web/statuses/106800377502349711)
- [Looks really good. The download button is way too big.](https://fosstodon.org/web/statuses/106800439199567580)
- [It looks great! A clean design is always so welcome. The first sentence tells the visitor what Tenacity does and then immediately gives easy access to the three top locations we'd want to visit next. Only then do the graphics and more in-depth detail follow. I'd go for a slightly more...dynamic(?) showcase graphic. If that's even possible for an audio editor! The features section looks slightly haphazard and is perhaps missing a feature: respect. "No opt-out tracking or telemetry".](https://fosstodon.org/web/statuses/106800450823759111) (Note: Audacity's tracking is opt-in now, by the way)
- [Clear, approachable, and zero learning curve! Well done sosasees :)](https://twitter.com/TimothysWriting/status/1429466384073953284)

**Reply by n0toose** [reply]
An interesting idea would be moving Key from the website to the web console instead!

**Reply by poyribn** [reply]
¿what's þe web console?
